lib/components/src/form/button.js

```diff
@@ -0,0 +1,10 @@
+import glamorous from 'glamorous';
+
+export default glamorous.button({
```

Comment:
Let's set `type="button"` here. By default buttons are `type="submit"`, but it's never what we actually want.

With glamorous, you can do it like that:
```
glamorous.button({
  <styles>
}).withProps({ type: 'button' })
```